datasets/dataforgood-fb-hrsl.yaml

```diff
@@ -0,0 +1,27 @@
+Name: High Resolution Population Density Maps + Demographic Estimates by CIESIN and Facebook
+Description: |
+  The population of the world, allocated to 1 arcsecond blocks. This
+  refines CIESIN’s Gridded Population of the World project, using
+  machine learning models on high-resolution worldwide Digital Globe
+  satellite imagery.
+Documentation: https://dataforgood.fb.com/tools/population-density-maps/
+Contact: disastermaps@fb.com
```

Comment:
Can you add a ManagedBy Field like `[Facebook](https://dataforgood.fb.com/)` or `[Facebook](https://facebook.com)`?